saleor/account/migrations/0068_user_uuid.py

```diff
@@ -14,6 +13,12 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name="user",
             name="uuid",
-            field=models.UUIDField(default=uuid.uuid4, unique=True),
+            field=models.UUIDField(default=None, null=True),
+        ),
+        migrations.RunSQL(
+            """
+            ALTER TABLE account_user ALTER COLUMN uuid SET DEFAULT null;
```

Comment:
It should be populated with UUID by default. This is useful in the case someone creates an user while we have 0069 applied but 0070 is not applied yet

```suggestion
            ALTER TABLE account_user ALTER COLUMN uuid SET DEFAULT gen_random_uuid();
```